src/Components/Open/SaveModelControl.jsx

```diff
@@ -191,19 +183,16 @@ function SaveModelDialog({isDialogDisplayed, setIsDialogDisplayed, navigate, org
       setRequestCreateFolder(false)
     }
 
-    setSelectedFolderName('none')
+    setSelectedFolderName('')
 
     setCurrentPath(newPath)
 
-    const {files, directories} = await getFilesAndFolders(repoName, owner, newPath, accessToken)
-    // eslint-disable-next-line no-shadow
-    const fileNames = files.map((file) => file.name)
```

Comment:
and there were some vars showing up in lint for me as unused, and these came out transitively as well